cmake/modules/FindCppCheck.cmake

```diff
@@ -0,0 +1,30 @@
+unset(CPPCHECK_BIN CACHE)
```

Comment:
at some point i wanted to test the negative case. so i uninstalled cppcheck, but since find_program cache the program, it did not detect the issue correctly unless i unset the variable first